README.md

```diff
@@ -5,796 +5,22 @@
 ![Scrutinizer][Scrutinizer Coverage Image]][Scrutinizer]
 [![SensioLabsInsight][SensioLabsInsight Image]][SensioLabsInsight]
 
-Table of contents:
-
-- [changelog](CHANGELOG.md)
-- [Installation](#installation)
-- [Basic usage and methods](#basic-usage)
-- [Command tests](#command-tests)
-- [Database and fixtures](#database-tests)
-- [Create an already logged client](#create-an-already-logged-client)
-- [Query Counter](#query-counter)
-- [Caveats](#caveats)
-- [paratest](paratest.md)
```

Comment:
It looks like links to `paratest.md` and `fastest.md` are missing. Could you please restore them?

Reply:
oh right, I need to move them too. Will do